### ISSUE TITLE
Could NOT find BISON 

### ISSUE THREAD

**Issue by l0o0** [vgteam/vg#701]
I've downloaded the git repo to build:
```bash
. ./source_me.sh
make
```
The output is 
> if [ ! -d bin ]; then mkdir -p bin; fi
if [ ! -d lib ]; then mkdir -p lib; fi
if [ ! -d obj ]; then mkdir -p obj; fi
if [ ! -d obj/unittest ]; then mkdir -p obj/unittest; fi
if [ ! -d obj/subcommand ]; then mkdir -p obj/subcommand; fi
if [ ! -d include ]; then mkdir -p include; fi
if [ ! -d cpp ]; then mkdir -p cpp; fi
cd deps/raptor/build && cmake .. && make && cp src/libraptor2.a /lustre/project/og02/linxingzhong/projects/Genome/KF-BJ-E-20151229-01_rice/08.evaluation/vg/vg-soft/vg.git/lib && mkdir -p /lustre/project/og02/linxingzhong/projects/Genome/KF-BJ-E-20151229-01_rice/08.evaluation/vg/vg-soft/vg.git/include/raptor2 && cp src/*.h /lustre/project/og02/linxingzhong/projects/Genome/KF-BJ-E-20151229-01_rice/08.evaluation/vg/vg-soft/vg.git/include/raptor2
-- Could NOT find CURL (missing:  CURL_LIBRARY CURL_INCLUDE_DIR) 
-- Could NOT find LibXml2 (missing:  LIBXML2_LIBRARIES LIBXML2_INCLUDE_DIR) 
-- Could NOT find LibXslt (missing:  LIBXSLT_LIBRARIES LIBXSLT_INCLUDE_DIR) 
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:108 (message):
  Could NOT find BISON: Found unsuitable version "2.4.1", but required is at
  least "3" (found /usr/bin/bison)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:313 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake/Modules/FindBISON.cmake:172 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:33 (FIND_PACKAGE)
-- Configuring incomplete, errors occurred!
See also "/lustre/project/og02/linxingzhong/projects/Genome/KF-BJ-E-20151229-01_rice/08.evaluation/vg/vg-soft/vg.git/deps/raptor/build/CMakeFiles/CMakeOutput.log".
make: *** [Makefile:292：lib/libraptor2.a] Error 1   

It shows bison version is 2.4.1 in `/usr/bin/bison`, but I have install a bison-3 in my home folder. After I run `bison --version`, it shows **bison (GNU Bison) 3.0.4**.  
How can I specify the bison for vg?

**Comment by JervenBolleman** [comment]
If you do ```which bison``` which one does it show?

**Comment by l0o0** [comment]
```bash
$ which bison
/nfs3/onegene/user/group1/linxingzhong/opt/bin/bison
```
The path is create by myself.

**Comment by JervenBolleman** [comment]
try
```
cd deps/bison
./autogen.sh
```

**Comment by JervenBolleman** [comment]
and then back to make in the vg dir

**Comment by l0o0** [comment]
@JervenBolleman bison doesn't exist under vg.git/deps.

**Comment by l0o0** [comment]
In readme, you seem to install vg in ubuntu. My server is Centos 6, is it possible to install vg in my server? I found it was quite hard to install dependents for vg in centos.

**Comment by JervenBolleman** [comment]
Yes that is possible but it is hard. See [this wiki page](https://github.com/vgteam/vg/wiki/Building-VG-on-Cent-OS-6.6-or-7--and-using-it)

When I said try above
I meant to say
cd deps/raptor
./autogen.sh

**Comment by l0o0** [comment]
@JervenBolleman Thanks for your help.

**Comment by l0o0** [comment]
Following the wiki, I go through the make process, but I get faied in the make test. 
> Test Summary Report
-------------------
t/05_vg_find.t     (Wstat: 0 Tests: 35 Failed: 16)
  Failed tests:  2-14, 30-32
t/06_vg_index.t    (Wstat: 0 Tests: 45 Failed: 23)
  Failed tests:  1, 5-6, 10-15, 18, 20-32
t/07_vg_map.t      (Wstat: 0 Tests: 33 Failed: 4)  Failed tests:  14, 18-19, 24
t/12_vg_kmers.t    (Wstat: 0 Tests: 16 Failed: 1)
  Failed test:  6
t/14_vg_mod.t      (Wstat: 0 Tests: 39 Failed: 3)
  Failed tests:  4-6
t/15_vg_surject.t  (Wstat: 0 Tests: 11 Failed: 1)
  Failed test:  7
t/19_vg_compare.t  (Wstat: 0 Tests: 1 Failed: 1)
  Failed test:  1
t/27_vg_genotype.t (Wstat: 0 Tests: 6 Failed: 5)
  Failed tests:  1-5
t/29_vg_locify.t   (Wstat: 0 Tests: 9 Failed: 9)
  Failed tests:  1-9

Is there any clue to fix the failed tests?

**Comment by adamnovak** [comment]
Can you paste the output from the individual failing tests (further up in the log)? It ought to show what result each test expected and what result it got.

Do you have `jq` installed? It's not a build dependency, but the tests use it extensively.

**Comment by faithokamoto** [comment]
We've had a lot of dependency and other changes in the time since. If this or a similar problem reoccurs, please open a new Issue with updated vg.
